build.cake

```diff
@@ -241,14 +282,16 @@ Task("BuildAll")
 Task("Pack")
     .IsDependentOn("BuildAll")
     .IsDependentOn("PackDotNetTemplates")
-    .IsDependentOn("PackVSTemplates")
     .IsDependentOn("PackVSMacTemplates");
 
 Task("Test")
     .IsDependentOn("TestWindowsDX")
     .IsDependentOn("TestDesktopGL")
     .IsDependentOn("TestTools");
 
+Task("PublishStable")
+    .IsDependentOn("PublishVSTemplates");
```

Comment:
@harry-cpp I'm not sure when we want to publish new templates. Every time the develop branch changes? Apart from this, this PR is ready.

Reply:
I think so, yea.

Reply:
Wait a second, are you talking about publishing vsix to teamcity or marketplace, the later should be done only on releases.

Reply:
Yeah seems better for this to be a Teamcity job to publish the template and not a cake script thing.

Reply:
@Jjagg Any chance you can update this?